### DISCUSSION TITLE
Add End-of-life (EOL) / End-of-support information for components

### DISCUSSION THREAD

**Discussion by hvardhan20** [DependencyTrack/dependency-track#3096] [Ideas]
Along with Out-of-date components, we should be able to get EOL information of components. This information is tracked by https://endoflife.date/

We could add an analyzer for https://endoflife.date/ to fetch this information for components that are tracked currently.

**Comment by setchy** [comment]
corresponding issue https://github.com/DependencyTrack/dependency-track/issues/3108
